MAUI/DataGrid/grid-events.md

```diff
@@ -80,6 +107,19 @@ private void dataGrid_CellLongPress(object sender, DataGridCellLongPressEventArg
 {% endhighlight %}
 {% endtabs %}
 
+### CellLongPressedCommand and its Parameter
+When we long presssed the cell, the `CellLongPressedCommand` binds to the command which you defined in your ViewModel. Additionally, we have the option to utilize the `CellLongPressedCommandParameter` for passing data to the Command upon its execution.
```

Comment:
The command will be invoked while long-pressing a cell in the SfDataGrid. The instance of `DataGridCellLongPressEventArgs` will be passed as the default value of the command's parameter. It can be customized by setting the `CellLongPressedCommandParameter` property.

Reply:
Changed